template.yaml

```diff
@@ -8,9 +8,9 @@ Metadata:
     Name: PersonData
     Description: Fetch person metadata by given parameters
     Author: Unit
-    SemanticVersion: 1.0.0
     SpdxLicenseId: MIT
     LicenseUrl: LICENSE
+    Labels: ['${CODEBUILD_RESOLVED_SOURCE_VERSION}', '${GIT_REPO}']
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Found an embedded parameter outside of an &#34;Fn::Sub&#34; at Metadata/AWS::ServerlessRepo::Application/Labels/0